lib/generators/react_on_rails/templates/base/base/config/initializers/react_on_rails.rb

```diff
@@ -42,4 +42,17 @@
   # React components.
   #
   config.server_bundle_js_file = "server-bundle.js"
+
+  ################################################################################
+  ################################################################################
+  # FILE SYSTEM BASED COMPONENT REGISTRY
+  ################################################################################
+  # components_directory is the directory that is used to automatically detect and register components for the usage on
+  # the rails view. By Default any component inside `ror_components` directory is registered for the usage on rails view
```

Comment:
1. default is nil
2. if defined:
  1. assets precompile will automatically run the rake task
  2. for development: react_component view for an `auto_load_bundle` == `true` will confirm that the generated file inside the `components_directory`, `generated/{component_name}` exists in the `packs` (entries) directory exists. If not, run the rake task to build, and log a message to the console that the generated files were rebuilt.

For non-development, raise an error suggesting that the rake task needs to run. This will mainly apply to a test run.